h2o-core/src/main/java/water/fvec/FileVec.java

```diff
@@ -39,6 +39,12 @@ public void setNChunks(int n){
     _nChunks = n;
     setChunkSize((int)length()/n);
   }
+
+  @Override
+  public int chunkLen(int cidx) {
```

Comment:
Please explain, this function is supposed to return the length of the chunk. In case of FileVec that is the byte-size of the chunk. The size of the chunk is predetermined before the parse. This override doesn't seem to do the right unless I am missing something.

Reply:
Damn, this is a leftover of my experiments. Sorry !